pyproject.toml

```diff
@@ -134,7 +134,26 @@ known-third-party = [
 [tool.ruff.per-file-ignores]
 "airflow/models/__init__.py" = ["F401"]
 "airflow/models/sqla_models.py" = ["F401"]
-
+"tests/providers/google/cloud/_internal_client/test_secret_manager_client.py" = ["E402"]
```

Comment:
Just curious, did these errors start coming only for 3.11?

Reply:
I think that was some kind of temporary problem after upgrading ruff at some point of time in older version of the PR. I removed it all and nothing happened. Good catch :).